sources/europe/at/TirisOrthofotoCIR.geojson

```diff
@@ -0,0 +1,954 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "tirol.gv.at-orthofoto",
```

Comment:
could you adjust this id so that it's unique (e.g. `tirol.gv.at-orthofoto-cir`)?